### DISCUSSION TITLE
How to increase row height for full screen display of Atomic Calendar

### DISCUSSION THREAD

**Discussion by rafal-re** [totaldebug/atomic-calendar-revive#1467] [Support]
Dear Community,

Forgive my newbie question (also could not find an answer in earlier discussion): 

I would like to display atomic calendar as one full screen card and therefore I need to make font and rows higher.
I installed card-mod and managed to increase font size (by putting font-size: 300%):
`card_mod:
  style: |
    ha-card {
      color: yellow;
      background-color: darkblue;
      font-size: 300%
    }
    .current-day {
      color: yellow !important
    }`
... but I struggle to increase accordingly the height of rows. Without that, the events with increased font size are overlapping.
I would appreciate any hint how to achieve it.
Thanks a lot in advance!

**Comment by BobBurgess** [comment]
Very keen for an answer to this, I am running into exactly the same problem! This is the same problem that I have identified here: 
https://github.com/totaldebug/atomic-calendar-revive/discussions/1483
